### PR TITLE
Added open source KanTV player

### PR DESCRIPTION
**Pull request by zhouwg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ Applications with support of IPTV streams.
 - [SS IPTV](https://play.google.com/store/apps/details?id=com.ssiptv.tvapp) - Free multi-purpose player for playing video in local network or through internet.
 - [Free WhatsUp TV](https://play.google.com/store/apps/details?id=au.com.deltaits.freewhatsuptv) - Player for IPTV (.m3u) playlists.
 - [TV Player by TechArt](https://play.google.com/store/apps/details?id=com.cy8018.tvplayer) - Open source ad-free IPTV client with language, country and category filtering and a quick list of favorite channels.
+- [kantv](https://github.com/zhouwg/kantv) - Open source Android/iOS online TV/radio/movie player based on FFmpeg n4.4, derived from original ijkplayer(https://github.com/bilibili/ijkplayer) , with some enhancements(https://github.com/zhouwg/kantv/blob/kantv/NEWS.md)
```
Comment (freearhey) [inline README.md:80]:
```suggestion
- [kantv](https://github.com/zhouwg/kantv) - Open source player for Android.
```

**Review by freearhey — CHANGES_REQUESTED** [review]
You can also add another link to the iOS section:

```
- [kantv](https://github.com/zhouwg/kantv) - Open source player for iOS.
```

**Comment by zhouwg** [comment]
> - [kantv](https://github.com/zhouwg/kantv) - Open source player for iOS

thanks your comment.

focus on target Android and very limited supportive for target iOS in project kantv at the moment. should I add another link to iOS section?

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
No, you don't have to.

**Review by zhouwg — COMMENTED** [review]
> No, you don't have to.

thanks.

**Pull request merged**